node/bin/src/replay_transport.rs

```diff
@@ -27,9 +28,29 @@ pub async fn replay_server(
             let (recv, mut send) = socket.split();
 
             let mut reader = BufReader::new(recv);
-            skip_http_headers(&mut reader)
+            let skipped_bytes = skip_http_headers(&mut reader)
                 .await
                 .expect("failed to skip HTTP headers");
+            let block_replay_query = if let Ok(skipped_text) = String::from_utf8(skipped_bytes) {
+                if let Some(url_str) = skipped_text
+                    .split_whitespace()
+                    .find(|s| s.starts_with("/block_replays"))
+                {
+                    if let Ok(url) = Url::parse(&format!("http://dummy{url_str}")) {
```

Comment:
We are slowly starting to implement an HTTP protocol parser at this point... I know this is not directly related to your PR but any idea where we are on being able to do raw TCP from the infra standpoint?